bloom-housing/bloom
# feat: unconfirmed user login error fix

## Issue Overview

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/3912)


## Description
- This makes it so an unconfirmed user on the public site attempting to login is prompted to resend the confirmation email

- this moves the password validity check earlier during login

- public users can no longer log into partner site

- adds spanish translations to seeding script

## How Can This Be Tested/Reviewed?
- create a user on the public site (do not confirm them) attempt to log in, you should now be prompted to resend confirmation email

- as a public only user you should no longer be able to log into the partner site (note locally you will be auto logged in if you are logged into the public site

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
